tests/src/test_helpers/ssl_helpers.c

```diff
@@ -224,7 +224,7 @@ int mbedtls_test_ssl_buffer_put( mbedtls_test_ssl_buffer *buf,
     }
 
     buf->content_length += input_len;
-    return input_len;
+    return ( input_len > INT_MAX ) ? INT_MAX : input_len;
```

Comment:
the logic is now correct, but you will probably need to cast `input_len` to `int`. Sorry, that's my fault for not including this in my Slack message. (And the same for other places)

```suggestion
    return ( input_len > INT_MAX ) ? INT_MAX : (int) input_len;
```

Reply:
Oh, my bad. I didn't realize this comment when I updated the changes. I'll force-push a new change to fix this. Thanks for letting me know.